### PR TITLE
issue-2-add-location | Samuel Cho

### PR DESCRIPTION
**Pull request by Samuel-Cho**
Issue 2: User can add a location. 

Screen Shot 2021-04-07 at 4 20 01 PM
Screen Shot 2021-04-07 at 4 20 09 PM

### CODE REVIEW

File: client/components/search-form.jsx [hunk 1]
```diff
@@ -12,7 +25,8 @@ export default class SearchForm extends React.Component {
               type="text"
               name="location"
               className="location-search"
-              placeholder="Location" />
+              placeholder="Location"
+              onChange={this.handleChange} />
```
Comment (thebearingedge) [inline client/components/search-form.jsx]: Your inputs need a `value` prop.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please see my note in the code. Also, please add more functionality to your form. At least when the user submits the form, it should switch screens and update the URL hash.

**Comment by Samuel-Cho** [comment]
Screen Shot 2021-04-08 at 11 19 04 AM
Screen Shot 2021-04-08 at 11 19 31 AM
Screen Shot 2021-04-08 at 11 20 21 AM

**Comment by Samuel-Cho** [comment]
Screen Shot 2021-04-08 at 12 42 33 PM
Screen Shot 2021-04-08 at 12 42 57 PM

File: client/components/search-form.jsx [hunk 2]
```diff
@@ -1,73 +1,162 @@
...
+        if (foodType.id === this.state.foodType) {
+          return (
+            <div onClick={this.handleClick} className="food-type selected" id={foodType.id} key={foodType.id}>{foodType.text}</div>
+          );
+        } else {
+          return (
+            <div onClick={this.handleClick} className="food-type" id={foodType.id} key={foodType.id}>{foodType.text}</div>
+          );
+        }
+      });
```
Comment (thebearingedge) [inline client/components/search-form.jsx]: Are you sure you need both of these? I think the second one with the `if` in the `.map()` is enough.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by Samuel-Cho** [comment]
Screen Shot 2021-04-08 at 1 15 54 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**